src/management/fs.c

```diff
@@ -430,6 +439,45 @@ static int fs_delete_process_request(struct controller_api *fs_delete,
 
 	log_debug("Deleting FS : %s.", fs_delete_api->req.fs_name);
 
+	if (fs_name_hash == NULL) {	
+		log_err("Object hash not sent");
+		rc = EINVAL;
+		request_set_errcode(request, rc);
+		fs_create_send_response(fs_delete, NULL);
+		goto error;
+	}
+
+	/* If we every become multi-threaded, then this wont be sufficient. */
```

Comment:
typo correction .. every -> ever.
What extra needs to be done for multi-threaded environment ? please mention that. Or else mention what will fail in multithreaded env

Reply:
Done.